app/controllers/annotations_controller.rb

```diff
@@ -122,25 +55,22 @@ def getAnnotationsForCanvasViaLists
       p  "in getAnnotationsForCanvasViaLists: lists.count = #{lists.count}"
 
       lists.each do |list|
-        #p  "               list_id = #{list.list_id}"
         layer_id = getLayerFromListName list.list_id
         if !layer_id.nil?
           annotations = ListAnnotationsMap.getAnnotationsForList list.list_id
           annotations.each do |annotation|
             if !annotation.nil? &&
               if !annotation.active==false # [jrl]
-                #p "annotation.active = #{annotation.active}"
                 annoWLayerHash= Hash.new
                 annoWLayerHash["layer_id"] = layer_id
                 annoWLayerHash["annotation"] = annotation.to_iiif
+                annoWLayerHash["tags"] = annotation.annotation_tags
```

Comment:
annoWLayerHash here is the JSON object that is going to be returned, which contains layer_id and annotation. We don't want to add another key "tag" here, but instead we should make sure that annoWLayerHash["annotation"] has the correct tag values in its "resource" field. For that you should probably modify Annotation.to_iiif .